<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerWidget</class>
 <widget class="QWidget" name="ColorPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPicker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_activate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/pick_48.png</normaloff>:/assets/pick_48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_lighter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_main">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_darker">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_main">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lighter">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_darker">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="magnifierCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show magnifier</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
